packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -378,6 +378,13 @@ const defaultConfigs = {
       { type: "cryptowatch", exchange: "bitfinex", pair: "amplusd" }
     ]
   },
+  CNYUSD: {
+    type: "tradermade",
+    pair: "CNYUSD",
+    minuteLookback: 7200,
```

Comment:
Curious: what's the lookback limit for the API for the hourly and minutely endpoints? We should add that as a comment somewhere in the TraderMadePriceFeed